acstools/polarization_tools.py

```diff
@@ -0,0 +1,414 @@
+
+    >>> print(tables.wfc_transmission.meta['description'])
+    WFC filters use MJD corresponding to 2020-01-01. HRC filters use MJD corresponding to 2007-01-01.
+    """
+    def __init__(self, input_dict):
+
+        self.data = input_dict
+
+        self.wfc_transmission = Table(self.data['transmission']['wfc'],
+                                      names=('filter', 't_para', 't_perp', 'correction'),
+                                      meta=self.data['transmission']['meta'])
+
+        self.hrc_transmission = Table(self.data['transmission']['hrc'],
+                                      names=('filter', 't_para', 't_perp', 'correction'),
+                                      meta=self.data['transmission']['meta'])
+
+        self.wfc_efficiency = Table(self.data['efficiency']['wfc'], names=('filter', 'pol0', 'pol60', 'pol120'),
+                                    meta=self.data['efficiency']['meta'])
+
+        self.hrc_efficiency = Table(self.data['efficiency']['hrc'], names=('filter', 'pol0', 'pol60', 'pol120'),
+                                    meta=self.data['efficiency']['meta'])
+
+    @classmethod
+    def from_yaml(cls, yaml_file):
+        """
+        Read in a YAML file containing polarizer calibration data.
+
+        Parameters
+        ----------
+        yaml_file : str
+            Path to the YAML file containing the polarizer calibration information.
+
+        Returns
+        -------
+        `~acstools.polarizer_tools.PolarizerTables` object
+        """
+        with open(yaml_file, 'r') as yf:
+            input_dict = yaml.safe_load(yf)
+        return cls(input_dict)
+
+    @classmethod
+    def from_package_data(cls):
+        """
+        Use the YAML file contained within the acstools package to retrieve the polarizer
+        calibration data.
+
+        Returns
+        -------
+        `~acstools.polarizer_tools.PolarizerTables` object
+        """
+        filename = get_pkg_data_filename(os.path.join('data', 'polarizer_tables.yaml'))
+        return cls.from_yaml(filename)
+
+
+class Polarization:
+    """
+    Class for handling ACS polarization data. Input data for this class come from
+    photometry of ACS polarization images. The methods associated with this class
+    will transform the photometry into Stokes parameters and polarization properties,
+    i.e., the polarization fraction and position angle.
+
+    Parameters
+    ----------
+    pol0 : float
+        Photometric measurement in POL0 filter. Units: electrons or electrons/second.
+    pol60 : float
+        Photometric measurement in POL60 filter. Units: electrons or electrons/second.
+    pol120 : float
+        Photometric measurement in POL120 filter. Units: electrons or electrons/second.
+    filter_name : str
+        Name of the filter crossed with the polarization filter, e.g., F606W.
+    detector : {'wfc', 'hrc'}
+        Name of the ACS detector used for the observation. Must be either WFC or HRC.
+    pav3 : float
```

Comment:
Make it slightly less sneaky? 😸 
```suggestion
    pav3 : float or `~astropy.units.Quantity`
```